web/package.json

```diff
@@ -9,6 +9,7 @@
   "private": true,
   "scripts": {
     "test:unit": "jest",
+    "postinstall": "cp -R node_modules/monaco-editor/min/vs/ ./public/vs",
```

Comment:
have to create copy monaco-editor files to public folder in order to load monaco-editor js files locally 
https://github.com/suren-atoyan/monaco-react/issues/168#issuecomment-762336713